websockets/cookies/support/set-cookie.py

```diff
@@ -1,6 +1,7 @@
 from six.moves.urllib import parse
 
+from wptserve.utils import isomorphic_encode
 
 def main(request, response):
-    response.headers.set('Set-Cookie', parse.unquote(request.url_parts.query))
-    return [("Content-Type", "text/plain")], ""
+    response.headers.set(b'Set-Cookie', isomorphic_encode(parse.unquote(request.url_parts.query)))
+    return [(b"Content-Type", b"text/plain")], ""
```

Comment:
We should have `b""` for the last value, right?

Reply:
Yes. code updated. Thank you!